skills/declarative/calendarSkill/dialogs/SearchEventsDialog/SearchEventsDialog.dialog

```diff
@@ -0,0 +1,373 @@
+          "name": "SearchEventIn"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
+          "$designer": {
+            "id": "ET8AYy"
+          },
+          "eventName": "CollectOptions"
+        },
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "wYIwoN"
+          }
+        },
+        {
+          "$kind": "Microsoft.IfCondition",
+          "$designer": {
+            "id": "ekNrAH"
+          },
+          "condition": "dialog.start == null && dialog.title == null && (dialog.personNameArray == null || count(dialog.personNameArray) == 0)",
+          "actions": [
+            {
+              "$kind": "Microsoft.TraceActivity",
+              "$designer": {
+                "id": "1eO2VL"
+              },
+              "name": "CollectOption"
+            },
+            {
+              "$kind": "Microsoft.DeleteProperties",
+              "$designer": {
+                "id": "7HgKDy"
+              },
+              "properties": [
+                "dialog.query"
+              ]
+            },
+            {
+              "$kind": "Microsoft.TextInput",
+              "$designer": {
+                "id": "a8RIpz"
+              },
+              "maxTurnCount": 3,
+              "alwaysPrompt": false,
+              "prompt": "${TextInput_Prompt_a8RIpz()}",
+              "invalidPrompt": "${TextInput_InvalidPrompt_a8RIpz()}",
+              "unrecognizedPrompt": "${TextInput_UnrecognizedPrompt_a8RIpz()}",
+              "property": "dialog.query",
+              "validations": [
+                "@datetime!=null || @title!=null || @personName!=null || #no != null"
+              ],
+              "allowInterruptions": "=#No!= null && turn.recognized.score > 0.8"
+            },
+            {
+              "$kind": "Microsoft.EmitEvent",
+              "$designer": {
+                "id": "B9EWPt"
+              },
+              "eventName": "CollectOptions"
+            }
+          ],
+          "elseActions": []
+        },
+        {
+          "$kind": "Microsoft.Graph.Calendar.GetEvents",
+          "$designer": {
+            "id": "QXHo4q"
+          },
+          "resultProperty": "dialog.events",
+          "token": "=user.token.token",
+          "titleProperty": "=coalesce(dialog.title, null)",
+          "locationProperty": "=coalesce(dialog.location, null)",
+          "attendeesProperty": "=coalesce(dialog.personNameArray, null)",
+          "startProperty": "=coalesce(dialog.start, formatDateTime(utcNow(), 'yyyy-MM-dd'))",
+          "endProperty": "=coalesce(dialog.end, null)",
+          "timeZoneProperty": "=user.workingHours.timeZone.name",
+          "isFutureProperty": "=coalesce(dialog.futureEventsOnly, false)",
+          "isFutureEventsOnlyProperty": "=dialog.futureEventsOnly",
+          "dateTimeTypeProperty": "= coalesce(dialog.type, \"date\")"
+        },
+        {
+          "$kind": "Microsoft.SetProperties",
```

Comment:
Why did you need to set this to null here?

Reply:
Removed for the latest flow. It was because when I repeated the dialog, the luis result would influence the flow.